plane/src/database/backend.rs

```diff
@@ -355,6 +382,15 @@ impl<'a> BackendDatabase<'a> {
         Ok(())
     }
 
+    pub async fn publish_metrics(&self, metrics: BackendMetricsMessage) -> sqlx::Result<()> {
+        emit_with_key(
```

Comment:
Currently, everything we emit is stored in the DB. We should have a way of emitting "ephemeral" messages that are not stored in the DB. Since that doesn't yet exist, I'm ok with merging this and then doing that after.

Reply:
Good call. Pushed a commit to address this. Let me know what you think